users/tests/conftest.py

```diff
@@ -326,7 +326,9 @@ def get_json(self, url, *args, **kwargs):
     get_loa = HelsinkiTunnistus.get_loa
 
 
-def start_oidc_authorize(django_client, oidcclient_factory, backend_name=DummyFixedOidcBackend.name, state=None):
+def start_oidc_authorize(
```

Comment:
It might be nicer to have a separate `extra_data` argument and not just put all remaining arguments to `authorize_data`. Or explicitly just add a `ui_locales` parameter.

Reply:
True. Added separate `ui_locales` argument.